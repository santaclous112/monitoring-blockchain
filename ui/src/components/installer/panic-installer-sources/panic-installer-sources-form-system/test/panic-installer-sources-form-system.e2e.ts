import {newSpecPage} from '@stencil/core/testing';
import {
  PanicInstallerSourcesFormSystem
} from "../panic-installer-sources-form-system";

describe('panic-installer-sources-form-system', () => {
  it('renders', async () => {
    const page = await newSpecPage(
        {
          components: [PanicInstallerSourcesFormSystem],
          html: '<panic-installer-sources-form-system></panic-installer-sources-form-system>',
        }
    );

    const element = page.body.querySelector('panic-installer-sources-form-system');
    expect(element).toMatchSnapshot();
  });
});
