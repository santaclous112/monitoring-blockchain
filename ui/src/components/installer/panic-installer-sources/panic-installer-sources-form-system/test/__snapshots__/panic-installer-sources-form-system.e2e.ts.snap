// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`panic-installer-sources-form-system renders 1`] = `
<panic-installer-sources-form-system>
  <svc-input label="System Name" lines="inset" name="name" placeholder="undefined_system_1" required=""></svc-input>
  <div class="panic-installer-sources-form__help-text">
    <span>
      This will be used to identify the current System configuration.
    </span>
  </div>
  <div class="panic-installer-sources-form__input_test_box">
    <svc-input label="Node Exporter URL" lines="inset" name="exporterUrl" placeholder="https://IP:9100/metrics" required="" type="url"></svc-input>
    <panic-installer-test-button service="node-exporter"></panic-installer-test-button>
  </div>
  <div class="panic-installer-sources-form__help-text">
    <span>
      This is the node exporter URL of your system.
    </span>
  </div>
  <svc-toggle checked="" label="Monitor System" lines="inset" name="monitor" value=""></svc-toggle>
  <div class="panic-installer-sources-form__help-text">
    <span>
      Whether to monitor this configured system.
    </span>
  </div>
</panic-installer-sources-form-system>
`;
